app/models/requirement.rb

```diff
@@ -92,11 +90,6 @@ def formio_type_options
     DEFAULT_FORMIO_TYPE_TO_OPTIONS[input_type.to_sym] || {}
   end
 
-  # TODO: requirement codes should be auto generated. Using a uuid for now. Later we have to replace it with a more human readable code
-  def set_requirement_code
-    self.requirement_code = SecureRandom.uuid
-  end
-
```

Comment:
This should probably be still included, as other requirements will need it, which are not seeded.

def set_requirement_code
    self.requirement_code ||= SecureRandom.uuid
  end

Reply:
Yup sure. I added to before validation in case we add validations related to the code